website/content/partials/builder/amazon/common/RunConfig-not-required.mdx

```diff
@@ -364,6 +364,8 @@
   password for Windows instances. Defaults to 20 minutes. Example value:
   10m
 
+- `metadata_options` (MetadataOptions) - Metadata Settings
```

Comment:
You can add a new section in the builders' documentation to show the config for this one. 

Like: 
```
#### Metadata Settings

@include 'builder/amazon/common/MetadataOptions.mdx'

@include 'builder/amazon/common/MetadataOptions-not-required.mdx'

// Some example 

```
I suggest adding after the `### Run Configuration` section